lib/puppet/provider/route53_record.rb

```diff
@@ -39,23 +47,48 @@ def exists?
   end
 
   def record_hash(action)
-    values = resource[:values] || @property_hash[:values]
+    values = resource[:values]
```

Comment:
Someone please tell me if this is bad.  I don't know why we would ever take the value from the discovered resource.